### DISCUSSION TITLE
Calendrier synchronisation

### DISCUSSION THREAD

**Discussion by thsfo** [kimai/kimai#3842] [General]
Hello,
I was wondering if there was a way to have a calendar output to display it on a smartphone? (Using an Opensource ecosystem like yunohost/nextcloud ... I don't really want to use Google Calendar for example)
Ideally, it would be amazing to be able to edit them as well... but I don't dare to dream about it? 
Do you have any experience in this area? 

Nice evening!

ps it would be great if you answered yes! This software is exactly the one that would suit my needs... just missing those calendar in/out and it's perfection!

**Comment by kevinpapst** [comment]
Please post in english
